utils/nctl/sh/scenarios/emergency_upgrade_test.sh

```diff
@@ -54,7 +54,7 @@ function main() {
             equivocators=0 \
             doppels=0 \
             crashes=5 \
-            restarts=15 \
+            restarts=20 \
```

Comment:
Can you please add a comment that `20` is probably not the expected value, but it is here temporarily, until the `fast-sync` is released?